cmd/dep/status.go

```diff
@@ -973,6 +1182,40 @@ func formatVersion(v gps.Version) string {
 	return v.String()
 }
 
+func formatDetailVersion(v gps.Version, r gps.Revision) rawDetailVersion {
+	if v == nil {
+		return rawDetailVersion{
+			Revision: r.String(),
+		}
+	}
+	switch v.Type() {
+	case gps.IsBranch:
+		return rawDetailVersion{
+			Branch:   v.String(),
```

Comment:
Hi @zjs,

Does this include the `"branch BRANCHNAME"` format when executed like this? If so, please check for that and reduce to just `BRANCHNAME`.

Reply:
This won't include the "branch" string; that's added by the `formatVersion` function above, which this method is called instead of. I'd rename that something like `formatBasicVersion`, but it's also used for the `old` and `missing` codepaths.

Reply:
Hi @zjs,

Gotcha. Just checking since we had that conversation earlier.